service/raft/raft_address_map.hh

```diff
@@ -112,24 +112,24 @@ class raft_address_map {
         // end.
         using list_type = bi::list<expiring_entry_ptr>;
 
-        explicit expiring_entry_ptr(list_type& l, timestamped_entry* e)
+        explicit expiring_entry_ptr(list_type& l, timestamped_entry& e)
             : _expiring_list(l), _ptr(e)
         {
-            _ptr->_last_accessed = Clock::now();
-            _ptr->set_lru_back_pointer(this);
+            _ptr._last_accessed = Clock::now();
```

Comment:
May be also rename `_ptr` to something else as well?

Reply:
This field gets removed in a later commit anyway